homeassistant/components/shelly/__init__.py

```diff
@@ -141,6 +149,8 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry):
         )
     )
     if unload_ok:
-        await hass.data[DOMAIN].pop(entry.entry_id).shutdown()
```

Comment:
This shutdown still exists?

Reply:
No, `device.shutdown()` has been removed [here](https://github.com/home-assistant-libs/aioshelly/commit/0b4dca5fe557ea848077a915f76978f2e4703846#diff-a329664887efd63ea2246815356b0379e2c33a6e0b78731997e35713f9541189L243-L245).

Reply:
This is the device wrapper, not the aioshelly device

Reply:
You're right I mixed up shutdowns, what a mess I made :facepalm: